src/KeyVault/Microsoft.Azure.KeyVault/Generated/Models/KeyBundle.cs

```diff
@@ -59,5 +63,11 @@ public KeyBundle(JsonWebKey key = default(JsonWebKey), KeyAttributes attributes
         [JsonProperty(PropertyName = "tags")]
         public IDictionary<string, string> Tags { get; set; }
 
+        /// <summary>
+        /// Gets or sets true if the secret's lifetime is managed by key vault.
```

Comment:
>/// Gets or sets true if the secret's lifetime is managed by key vault. [](start = 8, length = 71)

Consider mentioning an example. i.e. if this is a key backing a certificate, then managed will be true.